libkbfs/init.go

```diff
@@ -616,11 +616,13 @@ func doInit(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn,
 		log.Warning("Could not enable disk limiter: %+v", err)
 		return nil, err
 	}
+	goctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
`ctx` was taken by something that isn't a `context.Context`, so I was just going for a name that differentiated it. I'll change it to `ctx10s`.